### DISCUSSION TITLE
Should we release 0.67.2?

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#12] [Patches]
## 0.67.2 has been released. See discussion for [0.67.3](https://github.com/reactwg/react-native-releases/discussions/14)

Conversations on this thread are limited: 
* [major release issues](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking)
* [qualified cherry-pick requests](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://github.com/facebook/react-native/wiki/Release-FAQ#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue. 
If the issue is a [major release issue](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking), please reference the issue here.

---
#### List of qualified picks
 
1. https://github.com/facebook/react-native/commit/35bcf934b186e581d100d43e563044300759557f
2. https://github.com/facebook/react-native/commit/60cef850bd3fd12c32ee1196bd19a559592d1465
3. https://github.com/facebook/react-native/commit/507b05f4c02b46109f483a2b79c924a775fd7bd3
4. https://github.com/facebook/react-native/commit/d6db5c54640cef999b6922a10ceb85dede929b3f
5. https://github.com/facebook/react-native/commit/76a2cf3569571b943b4bcc6867e069338ff88f1f
6. https://github.com/facebook/react-native/commit/6239e2f5ce82f7c2e683eb4699b9ce3ff1b58ac5

#### Local commits to backport to main
1.
```

**Comment by zibs** [comment]
Could we cherry-pick https://github.com/facebook/react-native/commit/35bcf934b186e581d100d43e563044300759557f? Currently `npx react-native init <App>` (0.67.1) results in an unbuildable `yarn ios` for me due to this `find-node.sh` script exiting with a non-zero exit code during `FBReactNativeSpec` when building. I'm on an m1/nvm type of setup. Not sure if this is a universal issue, but this commit fixed my issue at least.

**Reply by idrissakhi** [reply]
I think if you remove default nvm version alias, build will work just fine.
nvm unalias default

**Reply by zibs** [reply]
That doesn't really seem to help - it causes the build to "succeed", but then metro doesn't bundle```error: Can't find the 'node' binary to build the React Native bundle.  If you have a non-standard Node.js installation, select your project in Xcode, find  'Build Phases' - 'Bundle React Native code and images' and change NODE_BINARY to an  absolute path to your node executable. You can find it by invoking 'which node' in the terminal.```

since that command effectively unsets node globally on my system.

**Reply by arunim2405** [reply]
@zibs https://github.com/facebook/react-native/pull/32955

**Reply by huynextlevel** [reply]
You can try with this to fixing `which node` issuse

```
sudo ln -s $(which node) /usr/local/bin/node
```

**Reply by kelset** [reply]
@zibs can you verify that cherry picking the commit would solve your issue by testing the artifact as explained by this comment? https://github.com/facebook/react-native/pull/32955#issuecomment-1019417339

**Reply by zibs** [reply]
Ah, thanks for the nudge @kelset.

It worked aside from some `yarn add` issues I was having...so in the end I just used `npm install --save <path to tarball>` and deleted my node modules, package-lock/yarn.lock and reinstalled, and everything built + rendered fine. 

@huynextlevel I've tried that, but it actually didn't seem to resolve my issue in this case. The other issue with that solution as well is if I'm running multiple RN projects which use different node versions things can get messy since the symlink can be pointing to a different node than the one I intend to use I believe...

**Reply by mlazari** [reply]
I have the same error after updating from `0.66.4` to `0.67.1`. Applying manually the fix from https://github.com/facebook/react-native/commit/35bcf934b186e581d100d43e563044300759557f in `node_modules/react-native/scripts/find-node.sh` fixes it for me, so I think that would solve the issue.

**Reply by ken0nek** [reply]
Same.

> Applying manually the fix from facebook/react-native@35bcf93 in node_modules/react-native/scripts/find-node.sh fixes it for me, so I think that would solve the issue.

This solved the issue.

**Comment by kelset** [comment]
Looks like there was a commit that broke Jest tests: https://github.com/facebook/react-native/issues/32939, apparently by looking at the PR it sounds like it should have been signaled as a breaking change: https://github.com/facebook/react-native/pull/32346

**Reply by lunaleaps** [reply]
How should this be handled? Are you suggesting a revert for 0.67?

**Reply by kelset** [reply]
The original author is looking into it, let's hear from him first - maybe there's a better option than just reverting. I'm not sure atm

**Reply by kelset** [reply]
PR with fix: https://github.com/facebook/react-native/pull/32964

**Comment by kelset** [comment]
We need this merged & cherry-picked: https://github.com/facebook/react-native/pull/32961

**Reply by tido64** [reply]
Commit: https://github.com/facebook/react-native/commit/60cef850bd3fd12c32ee1196bd19a559592d1465

**Comment by lunaleaps** [comment]
We should consider picking this https://github.com/facebook/react-native/commit/76a2cf3569571b943b4bcc6867e069338ff88f1f which updates our release automation to be less janky

**Comment by kelset** [comment]
we might have to cherry pick this https://github.com/facebook/react-native/commit/d6db5c54640cef999b6922a10ceb85dede929b3f to address the CVE

**Comment by enahum** [comment]
@kelset I want to propose the inclusion of https://github.com/facebook/react-native/pull/32989
